eth/backend.go

```diff
@@ -635,22 +639,60 @@ var (
 	ErrNotBorConsensus             = errors.New("not bor consensus was given")
 	ErrBorConsensusWithoutHeimdall = errors.New("bor consensus without heimdall")
 
-	whitelistTimeout = 30 * time.Second
+	whitelistTimeout      = 30 * time.Second
```

Comment:
could we extract it to Bor config?

Reply:
Both params

Reply:
Yes, But I think fixing it for all the validators would be better.